certificate/server.js

```diff
@@ -46,6 +46,7 @@ async function printPdf(name, trackingKey, udise) {
         donor: {
           trackingKey: trackingKey,
           name: name,
+          phone_number:phone_number,
```

Comment:
This will be a masked phone number - no point adding it to the certificate in this way. Let's skip this, and solve this in this [issue](https://github.com/Samagra-Development/x-admin/issues/2)